test/util/efficient_gen/edge_gen.cpp

```diff
@@ -2,7 +2,11 @@
 #include <algorithm>
 #include <boost/multiprecision/cpp_int.hpp>
 #include "efficient_gen.h"
-#include "../../../include/graph.h"
+
+enum UpdateType {
+  INSERT = 0,
+  DELETE = 1,
+};
```

Comment:
either get rid of this enum or the one in `graph.h`.

Reply:
The point of this enum was to remove the dependency on graph.h because that was causing the compilation to fail on my system.

Reply:
There are now two copies of this enum. There should only be one.